File: app/src/main/java/com/lc/bangumidemo/Activity/GlobalApplication.java
package com.lc.bangumidemo.Activity;

import android.app.Application;

import com.lc.bangumidemo.Green.DaoManager;

public class GlobalApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        initGreenDao();
    }

    private void initGreenDao() {
        DaoManager mManager = DaoManager.getInstance();
        mManager.init(this);
    }
}

File: app/src/main/java/com/lc/bangumidemo/Activity/GlobalApplication.kt
package com.lc.bangumidemo.Activity

import android.app.Application

import com.lc.bangumidemo.Green.DaoManager

class GlobalApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        initGreenDao()
    }

    private fun initGreenDao() {
        val mManager = DaoManager.getInstance()
        mManager.init(this)
    }
}
